nautobot/dcim/factory.py

```diff
@@ -60,7 +61,7 @@ class Meta:
     tenant = factory.Maybe("has_tenant", random_instance(Tenant), None)
 
     has_time_zone = factory.Faker("pybool")
-    time_zone = factory.Maybe("has_time_zone", pytz.timezone("US/Eastern"))
+    time_zone = factory.Maybe("has_time_zone", pytz.timezone(Faker().timezone()), None)
```

Comment:
I think this needs to be a lazyfunction, otherwise you're going to get the same timezone for every object